### DISCUSSION TITLE
Deploy in two or more clusters

### DISCUSSION THREAD

**Discussion by AkakievKD** [argoproj/argo-cd#9458] [Ideas]
Can one argoCD deploy in two or more clusters?

**Comment by AkakievKD** [comment]
And do you plane to add this feature?

**Comment by jgwest** [comment]
Hi @Nello-Angelo , Argo CD can deploy to multiple remote clusters, you need to ensure the cluster credentials are defined in Argo CD [for example](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#clusters) or via `argocd cluster add` CLI command.
